lib/portage/tests/process/test_PopenProcessBlockingIO.py

```diff
@@ -18,10 +18,7 @@
 
 class PopenPipeBlockingIOTestCase(TestCase):
     """
-    Test PopenProcess, which can be useful for Jython support:
```

Comment:
This is a bit misleading -- we should note why we're doing it, the tests are still the way they are, right?